host-controller/src/main/java/org/jboss/as/host/controller/ManagedServerBootCmdFactory.java

```diff
@@ -300,20 +299,17 @@ public HostControllerEnvironment getHostControllerEnvironment() {
             // Sets the initial log file to use
             command.add("-Dorg.jboss.boot.log.file=" + getAbsolutePath(new File(logDir), "server.log"));
 
-            // The default host controller and process controller configuration file
+            // Used as a fall back in case the default-server-logging.properties has been deleted
             final String domainConfigFile = "file:" + getAbsolutePath(environment.getDomainConfigurationDir(), "logging.properties");
-            // The configuration file from the system property, could the default domain/configuration/logging.properties file
-            final String systemPropConfigFile = WildFlySecurityManager.getPropertyPrivileged("logging.configuration", null);
             // The default configuration file to use if nothing is set
             final File defaultConfigFile = getAbsoluteFile(environment.getDomainConfigurationDir(), "default-server-logging.properties");
 
-            // Ignore the system property value if domain/configuration/logging.properties is used
-            if (domainConfigFile.equals(systemPropConfigFile) && defaultConfigFile.exists()) {
+            // If the default file exists, it should but we should be safe in case it was deleted by a user
+            if (defaultConfigFile.exists()) {
                 path = "file:" + defaultConfigFile.getAbsolutePath();
-            } else if (systemPropConfigFile != null) {
-                path = systemPropConfigFile;
             } else {
-                // Default to the domain/configuration/logging.properties if nothing else found
+                // Default to the domain/configuration/logging.properties if the default wasn't found to get initial
+                // boot logging
                 path = domainConfigFile;
```

Comment:
This should be handled better now. It will attempt to log an error if no configuration file could be found which may or may not work depending on how the HC logging is configured.

It also uses a file to convert the property to a real URL rather than attempting to create one with a simple `file:` prefix.